### DISCUSSION TITLE
Using v2ray in vscode

### DISCUSSION THREAD

**Discussion by huangwb8** [v2fly/v2ray-core#1457] [Q&A]
最近在学python爬虫，需要用到一个维基百科的地址，总是返回Timeout错误。
在chrome都是可以正常地访问目标网站的。
我觉得如果可以在vscode上设置合适的代理就行了。

经过搜索，我试过以下策略（在setting.json中设置）。
```
 "http.proxy": "http://127.0.0.1:1080"
 "http.proxy": "http://127.0.0.1:10808"
 "http.proxy": "http://127.0.0.1:3XXX"（具体忘了，说是v2ray常用的监听端口，反正也是不行）
```
有没有大佬可以支点招？谢谢了！

**Comment by kallydev** [comment]
https://stackoverflow.com/q/5620263/12171408

**Reply by huangwb8** [reply]
不好意思，您的推荐答案与vscode的v2ray设置没有任何关联。我的问题不是关于如何在python代码中设置proxy去访问地址。我的问题是关于如何在vscode中与v2ray进行交互。

**Reply by huangwb8** [reply]
https://keer.me/vscode-configuration-system-agent-install-golang-extension.html 
我需要的是这种类型的答案~不过此答案并不适用于我的情况。
谢谢您！

**Comment by jia-fie** [comment]
作者这个问题解决了么？我也是类似的情况，我要访问openai的api也是能打开网站，但是连接不上

**Comment by tanghuipang** [comment]
试试 127.0.0.1:10809  ?

**Comment by Rryowa** [comment]
This is how to route VSCode through v2rayn
![image](https://github.com/user-attachments/assets/141fa7f4-ee9f-43e5-b715-edb8ca3b2ecc)

![image](https://github.com/user-attachments/assets/e3ef9ad1-0d41-4780-9863-ff1e3492e3f4)

**Comment by 3247553980c** [comment]
V2rayN左下角给了端口, socks:10808;http:10809, 所以应该用"http.proxy": "http://127.0.0.1:10809"
